recipes/bertrand/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(bertrand REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} bertrand_package_test.cpp)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
this feature was added in 3.8 but im not sure 17 was a defined value until later

Reply:
3.8 is fine for C++17